client/my-sites/theme/main.jsx

```diff
@@ -94,23 +114,36 @@ const ThemeSheet = React.createClass( {
 		);
 	},
 
-	renderSectionNav( section ) {
+	renderSectionNav( currentSection ) {
 		const filterStrings = {
-			details: i18n.translate( 'Details', { context: 'Filter label for theme content' } ),
-			documentation: i18n.translate( 'Documentation', { context: 'Filter label for theme content' } ),
+			overview: i18n.translate( 'Overview', { context: 'Filter label for theme content' } ),
+			setup: i18n.translate( 'Setup', { context: 'Filter label for theme content' } ),
 			support: i18n.translate( 'Support', { context: 'Filter label for theme content' } ),
 		};
 
+		const { selectedSite, id, supportDocumentation } = this.props;
+		const siteSlug = selectedSite ? selectedSite.slug : '';
+
+		function navItem( section ) {
+			return(
+				<NavItem
+					path={ `/theme/${ id }/${ section }/${ siteSlug }` }
+					selected={ section === currentSection }>
+					{ filterStrings[ section ] }
+				</NavItem>
+			);
+		}
+
 		const nav = (
 			<NavTabs label="Details" >
-				<NavItem path={ `/theme/${ this.props.id }/details` } selected={ section === 'details' } >{ i18n.translate( 'Details' ) }</NavItem>
-				<NavItem path={ `/theme/${ this.props.id }/documentation` } selected={ section === 'documentation' } >{ i18n.translate( 'Documentation' ) }</NavItem>
-				<NavItem path={ `/theme/${ this.props.id }/support` } selected={ section === 'support' } >{ i18n.translate( 'Support' ) }</NavItem>
+				{ navItem( 'overview' ) }
+				{ supportDocumentation && navItem( 'setup' ) }
+				{ navItem( 'support' ) }
```

Comment:
Would be cool if this was done via `map()` on what's currently `validSections` inside `validateSection()` to de-dupe section visibility logic -- maybe by making a `getValidSections()` method?